modules/overviews/app/controllers/overviews/overviews_controller.rb

```diff
@@ -2,8 +2,8 @@ module ::Overviews
   class OverviewsController < ::Grids::BaseInProjectController
     include OpTurbo::ComponentStream
 
-    before_action :authorize
```

Comment:
@jonasjabari the `::Grids::BaseInProjectController` already has this before action, calling it here will result in two code executions.

https://github.com/opf/openproject/blob/17b5a0c94cf6ee0ebdbbd019c33353902337eee2/modules/grids/app/controllers/grids/base_in_project_controller.rb#L4